toolbox/githubctl/main.go

```diff
@@ -47,9 +47,12 @@ var (
 	// needs to be consistent with prow config map
 	postSubmitJobs = []string{
```

Comment:
Why not read from the config map direclty ?

Reply:
https://github.com/kubernetes/test-infra/blob/master/prow/config/config.go#L263

Reply:
Thanks for the pointer and that is exactly what I plan to do too. This PR tries to unblock release while the work on reading config is pending. @rkpagadala has another PR this does this as well and he takes care of branches too so lets proceeds with his.